grafana-plugin/src/state/rootBaseStore/index.ts

```diff
@@ -80,29 +80,29 @@ export class RootBaseStore {
 
   // --------------------------
 
-  userStore: UserStore = new UserStore(this);
-  cloudStore: CloudStore = new CloudStore(this);
-  directPagingStore: DirectPagingStore = new DirectPagingStore(this);
-  grafanaTeamStore: GrafanaTeamStore = new GrafanaTeamStore(this);
-  alertReceiveChannelStore: AlertReceiveChannelStore = new AlertReceiveChannelStore(this);
-  outgoingWebhookStore: OutgoingWebhookStore = new OutgoingWebhookStore(this);
-
-  outgoingWebhook2Store: OutgoingWebhook2Store = new OutgoingWebhook2Store(this);
-  alertReceiveChannelFiltersStore: AlertReceiveChannelFiltersStore = new AlertReceiveChannelFiltersStore(this);
-  escalationChainStore: EscalationChainStore = new EscalationChainStore(this);
-  escalationPolicyStore: EscalationPolicyStore = new EscalationPolicyStore(this);
-  teamStore: TeamStore = new TeamStore(this);
-  telegramChannelStore: TelegramChannelStore = new TelegramChannelStore(this);
-  slackStore: SlackStore = new SlackStore(this);
-  slackChannelStore: SlackChannelStore = new SlackChannelStore(this);
-  heartbeatStore: HeartbeatStore = new HeartbeatStore(this);
-  scheduleStore: ScheduleStore = new ScheduleStore(this);
-  userGroupStore: UserGroupStore = new UserGroupStore(this);
-  alertGroupStore: AlertGroupStore = new AlertGroupStore(this);
-  resolutionNotesStore: ResolutionNotesStore = new ResolutionNotesStore(this);
-  apiTokenStore: ApiTokenStore = new ApiTokenStore(this);
-  globalSettingStore: GlobalSettingStore = new GlobalSettingStore(this);
-  filtersStore: FiltersStore = new FiltersStore(this);
```

Comment:
typing here is redundant. these attributes are properly typed w/o the explicit typing.